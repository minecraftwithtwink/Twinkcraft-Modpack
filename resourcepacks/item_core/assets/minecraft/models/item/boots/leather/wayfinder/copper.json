{
    "parent": "minecraft:item/generated",
    "textures": {
        "layer0": "minecraft:item/boots/base/leather/wayfinder",
        "layer1": "minecraft:item/boots/base/leather_overlay/wayfinder",
        "layer2": "minecraft:trims/items/boots_trim/wayfinder_copper"
    }
}