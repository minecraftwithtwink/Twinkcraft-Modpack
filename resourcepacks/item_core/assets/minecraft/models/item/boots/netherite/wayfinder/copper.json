{
    "parent": "minecraft:item/generated",
    "textures": {
        "layer0": "minecraft:item/boots/base/netherite/wayfinder",
        "layer1": "minecraft:trims/items/boots_trim/netherite/wayfinder_copper"
    }
}