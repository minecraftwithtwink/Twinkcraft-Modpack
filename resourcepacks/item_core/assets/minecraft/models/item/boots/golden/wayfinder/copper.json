{
    "parent": "minecraft:item/generated",
    "textures": {
        "layer0": "minecraft:item/boots/base/golden/wayfinder",
        "layer1": "minecraft:trims/items/boots_trim/wayfinder_copper"
    }
}